cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -1690,4 +1690,10 @@ void captured_lambda(int x, int &y, int &&z)
     };
 }
 
+int global_1;
+
+int global_2 = 1;
+
+const int global_3 = 2;
```

Comment:
My personal hobby horse I guess: a test case for a structured binding would also be welcome (yes these can occur globally outside functions).